### DISCUSSION TITLE
Attack in distributed mode

### DISCUSSION THREAD

**Discussion by dalqattan** [alibaba/FederatedScope#612] [Ideas]
Can we run an attack in distributed mode?
If yes, where can we add attack configuration in client config file or server config file?

Any thought!

Thanks
Duaa

**Comment by Osier-Yi** [comment]
> Can we run an attack in distributed mode? If yes, where can we add attack configuration in client config file or server config file?
> 
> Any thought!
> 
> Thanks Duaa 

Thanks for your interest in our attack module. Currently, our attack module only supports stand alone mode, and distributed mode is not supported yet.
